save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -216,69 +208,8 @@ class DemoManagerController(
             .bodyToMono()
     }
 
-    @GetMapping("/{organizationName}/{projectName}/status")
```

Comment:
and what's was wrong with these endpoints?

Reply:
This endpoints forward requests to demo's api part. No Permission check is required for it so there is better to send requests to save-demo directly